st2common/st2common/content/aliasesregistrar.py

```diff
@@ -43,8 +43,10 @@ def register_aliases_from_packs(self, base_dirs):
         :return: Number of aliases registered.
         :rtype: ``int``
         """
-        registered_count = 0
+        # Register packs first
+        self.register_packs(base_dirs=base_dirs)
```

Comment:
Yeah, sadly I couldn't come up with an approach which is much better than this.

We do have a base class with some shared code for resource registrars now, but sadly each class still duplicates a lot of code.